src/app/account/account/account.component.html

```diff
@@ -32,43 +38,96 @@ <h4>Corporation Information</h4>
+                <th>Facility Size</th>
+                <th>Units</th>
+                <th>Division</th>
+                <th class="actions">Actions</th>
+            </tr>
+            <tr *ngFor="let facility of facilityList">
+                <td>{{facility.name}}</td>
+                <td>{{facility.country}}</td>
+                <td>{{facility.type}}</td>
+                <td>{{facility.tier}}</td>
+                <td>{{facility.size}}</td>
+                <td>{{facility.units}}<span class="superscript">2</span></td>
+                <td>{{facility.division}}</td>
+                <td class="actions">
+                    <i class="fa fa-edit" title="Edit" (click)="setEditFacilityEntry(facility)"></i>
+                    <i class="fa fa-trash-o" title="Delete" (click)="setDeleteFacilityEntry(facility)"></i>
+                </td>
+            </tr>
+        </table>
+    </div>
     <button class="btn btn-link" (click)="addNewFacility()"><i class="fa fa-plus"></i> Add New Facility</button>
+
+    <br>
+    <br>
+    <br>
+    <br>
+
+    <div class="row">
+        <div class="col-lg-6">
+            <h4>Delete Account</h4>
+            <p>Once deleted, you will permanently lose all your data linked to your account. Please make a backup of your data before proceeding with this step.</p>
+            <button class="btn btn-danger" (click)="editAccount()">Delete Account</button>
+        </div>
+        <div class="col-lg-6">
+            <h4>Backup Account</h4>
+            <p>You can protect your files from accidental loss by creating a backup. We recommend backing up your data frequently.</p>
+            <button class="btn btn-outline">Backup Account</button>
+            <p class="small"> Last Backup: 12/04/2020</p>
+        </div>
+    </div>
+</div>
+
+<div class="window" [class.open]="facilityToEdit">
+    <div class="windowOverlay"></div>
+    <div class="windowContent">
+        <app-edit-facility *ngIf="facilityToEdit" [facility]="facilityToEdit" (emitClose)="closeEditFacility()"></app-edit-facility>
+    </div>
+</div>
+
+<div class="popup" [class.open]="showDeleteAccount">
+    <div class="popup-header">Delete Account: {{accountForm.controls.name.value}}
+        <button class="item-right" (click)="cancelDelete()">x</button>
+    </div>
+    <div class="popup-body">
+        <p>WARNING: By deleting this account, you are also deleting all of the data associated with it.
+            Are you sure you want to preform this action?</p>
+    </div>
+    <div class="popup-footer text-right">
+        <button class="btn btn-secondary" (click)="cancelAccountDelete()">Cancel</button>
+        <button class="btn btn-danger" (click)="confirmAccountDelete()">Confirm Delete</button>
+    </div>
+</div>
+
+<div class="popup" [class.open]="facilityToDelete" *ngIf="facilityToDelete">
+    <div class="popup-header">Delete Facility: {{facilityToDelete.name}}
+        <button class="item-right" (click)="cancelDelete()">x</button>
```

Comment:
cancelDelete() doesnt exist. Think it should be cancelFacilityDelete()

same above for cancelAccountDelete()